test/NuGet.Core.Tests/NuGet.ProjectModel.Test/PackageSpecOperationsTests.cs

```diff
@@ -253,6 +253,102 @@ public void AddOrUpdateDependency_ToSpecificFrameworks_AddsNewDependency()
                 spec.TargetFrameworks[1].Dependencies[0].LibraryRange.VersionRange.MinVersion);
         }
 
+        [Fact]
```

Comment:
All the modifications in this file are already in review based on @jeffkl  PR (https://github.com/NuGet/NuGet.Client/pull/4642) so I will remove these changes once his PR is merged into dev.